decidim-core/lib/decidim/core/engine.rb

```diff
@@ -3,6 +3,8 @@
 require "rails"
 require "active_support/all"
 require "action_view/railtie"
+require "active_model"
```

Comment:
Why do we need to add `active_model` here? Shouldn't it be loaded by the railtie?
https://github.com/rails/rails/blob/d38ff07a39c98f5db2b15ca3a51acee48c972205/activerecord/lib/active_record/railtie.rb#L6

https://github.com/rails/rails/blob/d38ff07a39c98f5db2b15ca3a51acee48c972205/activemodel/lib/active_model/railtie.rb#L3

Reply:
For some reason, if we do not explicitly require active model, the test suite is failing with `uninitialized constant ActiveModel::Validations::EachValidator`

Maybe is something else in the test suite that causes that, but it may be a subject to another PR, addressing explicitly only that topic.

Reply:
Could you try changing this:
https://github.com/decidim/decidim/blob/fc6e578c7a848a7ee3fa3ad8e5e15294120bc31e/decidim-dev/lib/decidim/dev/railtie.rb#L3

To:
```ruby
require "decidim/rails"
```

Reply:
@ahukkanen latest requested changes have been implemented. Can you have a look ?

Reply:
Thanks @alecslupu !

I actually meant could we change the line at `decidim/dev` railtie, you changed it at the core gem. But I don't think this would have any negative impact in real life, so for me it's fine as you changed it.

The only occasion where I can think this could have an impact (on load time) is when you only want to load the `Decidim::Core::Engine` outside of Rails but I cannot see when that would happen as the engine is only meant to be used with Rails.

Reply:
@ahukkanen I have implemented the required change as well, i have seen it after i already added my previous comment.

Reply:
It's kind of redundant to change it in both places but it's fine for me.

The `require "rails"` on top of each engine is basically to get access to `::Rails::Engine` which is needed to define the Engine. The added `require "decidim/rails"` does more than that, it requires all the engines Decidim needs to function properly.

But as said, it's fine as is. When we use `require`, it will be only loaded once, even if the same require statement appears multiple times.